modules/internal/ChapelLocale.chpl

```diff
@@ -159,31 +162,10 @@ module ChapelLocale {
 
     proc deinit() { }
 
-    inline proc maxTaskPar { return this._value.maxTaskPar; }
-    inline proc callStackSize { return this._value.callStackSize; }
-
-    // the following are normally taken care of by `forwarding`. However, they
-    // don't work if they are called in a promoted expression. See 15148
```

Comment:
I think it would be nice if we keep the comment here somehow.  If forwarding with promotion works, I wonder if it would allow us to just remove the functions as I hoped, or could we just remove forwarding and keep all these wrappers for the documentation's sake.

Reply:
I added a second copy of it above the primary methods (the first had moved down with the secondary method it preceded).